test/test.js

```diff
@@ -76,6 +76,19 @@ describe('cacheman-file', function() {
     });
   });
 
+  it('should sanitize key', function(done) {
```

Comment:
This test doesn't exactly show that the key was sanitized. It more so shows the functionality is still there. Maybe check to see how the key is stored or what the key looks like after it's been sanitized.